javascripts/discourse/api-initializers/discourse-mermaid-theme-component.js

```diff
@@ -1,7 +1,63 @@
-import Session from "discourse/models/session";
 import loadScript from "discourse/lib/load-script";
 import { apiInitializer } from "discourse/lib/api";
 
+async function applyMermaid(element, key) {
+  const mermaids = element.querySelectorAll("pre[data-code-wrap=mermaid]");
+
+  if (!mermaids.length) {
+    return;
+  }
+
+  await loadScript("https://unpkg.com/mermaid@8.13.10/dist/mermaid.min.js");
```

Comment:
At some point we'll have to change it so that we vendor the package in the component.